dll/win32/shell32/precomp.h

```diff
@@ -166,12 +194,28 @@ static inline UINT SeeFlagsToCmicFlags(UINT flags)
 
 // CStubWindow32 --- The owner window of file property sheets.
 // This window hides taskbar button of property sheet.
+#define CSTUBWINDOW32_CLASSNAME _T("StubWindow32")
 class CStubWindow32 : public CWindowImpl<CStubWindow32>
 {
 public:
-    DECLARE_WND_CLASS_EX(_T("StubWindow32"), 0, COLOR_WINDOWTEXT)
+    DECLARE_WND_CLASS_EX(CSTUBWINDOW32_CLASSNAME, 0, COLOR_WINDOWTEXT)
+    enum {
+        TYPE_FORMATDRIVE = 1,
+        TYPE_PROPERTYSHEET,
+    };
+    static LPCWSTR GetTypePropName() { return L"StubType"; }
```

Comment:
This is similar to what Windows shell has too?

Reply:
Windows does not have this property, no. I'm not sure how Windows does this. ROS already used a property for the format drive stub window, I just changed the name to make it more generic so every stub window user can use the same code.